Makefile

```diff
@@ -295,6 +295,16 @@ test_cmath: test/test_cmath.cc $(ALLHEADERS)
 	$(info 4: $(CXX) -I$(SRCINCDIR) $(CXXFLAGS) -std=c++11 $< -o test/$@_cxx11)
 	@-$(CXX) -I$(SRCINCDIR) $(CXXFLAGS) -std=c++11 $< -o test/$@_cxx11 &> /dev/null && echo "4: c++11 legacy cmath build success (test succeeded)!" || echo "4: c++11 legacy cmath build failure (test failed)!"
 
+# Special clause for testing faccessat in a setuid program.
+# Only counts as a test when run by root.
+# Assumes there is a _uucp user.
+# Tests setuid _uucp, setuid root, and setgid tty.
+test_faccessat_setuid: test/test_faccessat /opt/local/lib/libMacportsLegacySupport.dylib
```

Comment:
Why you haven't use `realpath` and `$BUILDSYSLIBPATH`? It allows to test against library which is build in the repo.

Reply:
Thanks for spotting that. The `/opt/local/lib/libMacportsLegacySupport.dylib` is no longer needed, now that the setuid test can test the library-being-tested rather than an irrelevant already-installed-library.